ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/RejectionReason.scala

```diff
@@ -7,7 +7,7 @@ package com.daml.ledger.participant.state.v1
   *
   * Used to provide details for [[Update.CommandRejected]].
   */
-sealed trait RejectionReason extends Product with Serializable {
+sealed abstract class RejectionReason extends Product with Serializable {
```

Comment:
why the change?

Reply:
it is the only one of many siblings that is a trait; abstract class is preferred due to performance on JVM